src/vs/workbench/api/electron-browser/mainThreadWebview.ts

```diff
@@ -200,9 +200,9 @@ export class MainThreadWebviews extends Disposable implements MainThreadWebviews
 			return;
 		}
 
-		const targetGroup = this._editorGroupService.getGroup(viewColumnToEditorGroup(this._editorGroupService, showOptions.viewColumn));
+		const targetGroup = this._editorGroupService.getGroup(viewColumnToEditorGroup(this._editorGroupService, showOptions.viewColumn)) || this._editorGroupService.getGroup(webview.group || 0);
```

Comment:
I'm not what the reasoning behind https://github.com/Microsoft/vscode/commit/2e6db8d955046349a941ab814183cf480b0e274b is, using `webview.group !== undefined ? webview.group : ACTIVE_GROUP` seems closer to the original logic

But since that's already in this seems fine

Reply:
we won't get any group by `ACTIVE_GROUP` (which is `-1`) so here I use `0` to ensure that the webview is revealed in the first view column if possible. @mjbvz can decide what's the best fix for it later.